backend/controller/dal/dal.go

```diff
@@ -1164,6 +1164,11 @@ func (d *DAL) InsertCallEvent(ctx context.Context, call *CallEvent) error {
 	}))
 }
 
+func (d *DAL) DeleteOldEvents(ctx context.Context, eventType EventType, age time.Duration) (int64, error) {
+	count, err := d.db.DeleteOldEvents(ctx, age, eventType)
+	return count, dalerrs.TranslatePGError(err)
```

Comment:
I'd add an exclusion for this lint error - we know it's already wrapped. Add it to .golangci.yml